data/jobs/2022-05-15-vondelft-xchem-compchem-postdoc.yaml

```diff
@@ -0,0 +1,27 @@
+role: To coordinate structure biology support and structural enablement across projects and cores
```

Comment:
There seems to be a mismatch with some ads treating `role` as a noun `Researcher in....` vs these which are more the infinitive `To ...`

I'd suggest we pick one style. After building the site locally. These long phrases `To coordinate structure biology support and structural enablement across projects and cores` do seem a bit of a strange way to start the ad. 
<img width="849" alt="Screen Shot 2022-05-15 at 9 35 35 PM" src="https://user-images.githubusercontent.com/10458975/168505329-f8e0c5d3-1e0b-47b7-b902-387b36b05cea.png">

I'd suggest a change to something like "Researcher coordinating structure biology support and structural enablement across projects and cores"

Reply:
We should clean this up to make it consistent, but we might have to do it in a second pass.

We just need bare minimum functionality for now.